bftclient/include/bftclient/fake_comm.h

```diff
@@ -133,8 +133,20 @@ class FakeCommunication : public bft::communication::ICommunication {
   BehaviorThreadRunner runner_;
   std::thread fakeCommThread_;
 };
+std::vector<uint8_t> replayConfig(const MsgFromClient& msg);
 
 inline void immideateBehaviour(const MsgFromClient& msg, IReceiver* client_receiver) {
+  std::vector<uint8_t> reply = replayConfig(msg);
+  client_receiver->onNewMessage(msg.destination.val, reinterpret_cast<const char*>(reply.data()), reply.size());
+}
+
+inline void delayedBehaviour(const MsgFromClient& msg, IReceiver* client_receiver) {
+  std::vector<uint8_t> reply = replayConfig(msg);
+  std::this_thread::sleep_for(100ms);
+  client_receiver->onNewMessage(msg.destination.val, reinterpret_cast<const char*>(reply.data()), reply.size());
+}
+
+std::vector<uint8_t> replayConfig(const MsgFromClient& msg) {
```

Comment:
This needs to be marked `inline` to prevent ODR violations. This will fix the clang-tidy error as well.

Reply:
Fixed it! Thank you!